rest/data/commit_queue.go

```diff
@@ -318,6 +318,28 @@ func EnqueuePRToCommitQueue(ctx context.Context, env evergreen.Environment, sc C
 		return nil, errors.New("PR contains no base branch label")
 	}
 
+	if !thirdparty.IsUnblockedGithubStatus(*pr.MergeableState) {
+		errMsg := fmt.Sprintf("PR is not mergeable, status: %s", *pr.MergeableState)
+		grip.Error(message.Fields{
```

Comment:
I don't think we should log this at the error level, since it's not an evergreen error but rather information to help debug for users.

Reply:
done

Reply:
I think this change may have been left out